pandas/tests/groupby/test_groupby.py

```diff
@@ -1813,15 +1813,15 @@ def test_groupby_level(self):
         assert result1.index.name == frame.index.names[1]
 
         # groupby level name
-        result0 = frame.groupby(level='first').sum()
-        result1 = frame.groupby(level='second').sum()
+        result0 = frame.groupby(level='first', sort=False).sum()
+        result1 = frame.groupby(level='second', sort=False).sum()
         assert_frame_equal(result0, expected0)
         assert_frame_equal(result1, expected1)
 
         # axis=1
 
-        result0 = frame.T.groupby(level=0, axis=1).sum()
-        result1 = frame.T.groupby(level=1, axis=1).sum()
```

Comment:
for a couple of these that you changed can you also add the sort=True case (maybe parametrize on sort=)?